rlbot_gui/gui/js/botmanager.js

```diff
@@ -0,0 +1,154 @@
+var globalJson
+
+	async function downloadRepos(){
+		await fetch('https://raw.githubusercontent.com/ard1998/RLBot-repos/master/stableThrusted.json')
+		.then(function(response) {
+			return response.json();
+		})
+		.then(function(myJson) {
+			globalJson = myJson;
+			addPackageData(myJson);
+		});
+	}
+
+	async function addPackageData(json){
+		for (var index = 0; index < json.repos.length; index++) {
```

Comment:
please be consistent in bracket spacing :)

Reply:
I will look for it :)